webapp/src/DataFixtures/DefaultData/LanguageFixture.php

```diff
@@ -23,30 +23,31 @@ public function __construct(
+            ['adb',    'ada',        'Ada',         ['adb', 'ads'],              false, null,         false,  true,   1,     'adb',               '',                   ''],
+            ['awk',    'awk',        'AWK',         ['awk'],                     false, null,         false,  true,   1,     'awk',               '',                   ''],
+            ['bash',   'bash',       'Bash shell',  ['bash'],                    false, 'Main file',  false,  true,   1,     'bash',              '',                   ''],
+            ['c',      'c',          'C',           ['c'],                       false, null,         true,   true,   1,     'c',                 'gcc --version',      ''],
+            ['cpp',    'cpp',        'C++',         ['cpp', 'cc', 'cxx', 'c++'], false, null,         true,   true,   1,     'cpp',               'g++ --version',      ''],
+            ['csharp', 'csharp',     'C#',          ['csharp', 'cs'],            false, null,         false,  true,   1,     'csharp',            '',                   ''],
+            ['f95',    'f95',        'Fortran',     ['f95', 'f90'],              false, null,         false,  true,   1,     'f95',               '',                   ''],
+            ['hs',     'haskell',    'Haskell',     ['hs', 'lhs'],               false, null,         false,  true,   1,     'hs',                '',                   ''],
+            ['java',   'java',       'Java',        ['java'],                    false, 'Main class', true,   true,   1,     'java_javac_detect', 'javac -version',     'java -version'],
+            ['js',     'javascript', 'JavaScript',  ['js'],                      false, 'Main file',  false,  true,   1,     'js',                '',                   ''],
+            ['lua',    'lua',        'Lua',         ['lua'],                     false, null,         false,  true,   1,     'lua',               '',                   ''],
+            ['kt',     'kotlin',     'Kotlin',      ['kt'],                      true,  'Main class', false,  true,   1,     'kt',                'kotlinc -version',   'kotlin -version'],
+            ['pas',    'pascal',     'Pascal',      ['pas', 'p'],                false, 'Main file',  false,  true,   1,     'pas',               '',                   ''],
+            ['pl',     'pl',         'Perl',        ['pl'],                      false, 'Main file',  false,  true,   1,     'pl',                '',                   ''],
+            ['plg',    'prolog',     'Prolog',      ['plg'],                     false, 'Main file',  false,  true,   1,     'plg',               '',                   ''],
+            ['py3',    'python3',    'Python 3',    ['py'],                      false, 'Main file',  true,   true,   1,     'py3',               'pypy3 --version',    'pypy3 --version'],
+            ['ocaml',  'ocaml',      'OCaml',       ['ml'],                      false, 'Main file',  false,  true,   1,     'ocaml',             'ocamlopt --version', ''],
```

Comment:
From the compile command it seems that Ocaml doesn't have any specific entry point, so I think this should be:
```suggestion
            ['ocaml',  'ocaml',      'OCaml',       ['ml'],                      false, null       ,  false,  true,   1,     'ocaml',             'ocamlopt --version', ''],
```